draft-ietf-jsonpath-base.md

```diff
@@ -1081,10 +1084,21 @@ Notes:
 * The behaviour of operators is consistent with the 'C'-family of programming languages.
 <!-- need to clarify -->
 
+The following table lists filter expression operators in order of precedence from highest to lowest.
+
+| Precedence | Operator type | Syntax |
+|:--:|:--:|:--:|
+|  5  | Grouping | `(...)` |
+|  4  | Logical NOT | `!` |
+|  3  | Relations | `==`<br>`!=`<br>`<`<br>`>`<br>`<=`<br>`>=`<br>`=~`<br>` in ` |
```

Comment:
Is it necessary to have each operator on its own line?  Seems like we could do with two or three per line.  Just a readability question.

Reply:
Not really. I've pushed a change which groups similar operators together on single lines.

Reply:
Perhaps I realise why this had one operator per line. Putting multiple on one line, separated by space or `&nbsp;` results in operators being wrapped in double quotes in the rendered text version (although not the HTML), thus:  
```
            +============+===========+===================+
              | Precedence |  Operator |       Syntax      |
              |            |    type   |                   |
              +============+===========+===================+
              |     5      |  Grouping |       (...)       |
              +------------+-----------+-------------------+
              |     4      |  Logical  |         !         |
              |            |    NOT    |                   |
              +------------+-----------+-------------------+
              |     3      | Relations |     "==" "!="     |
              |            |           | "<" "<=" ">" ">=" |
              |            |           |        "=~"       |
              |            |           |        "in"       |
              +------------+-----------+-------------------+
              |     2      |  Logical  |         &&        |
              |            |    AND    |                   |
              +------------+-----------+-------------------+
              |     1      |  Logical  |         ||        |
              |            |     OR    |                   |
              +------------+-----------+-------------------+
```
This is pretty ugly, so I'm going to revert to one operator per line until someone can suggest better markup.

Reply:
Huh! The double quotes are there in the text version even when there is only one operator per line. Given I can't remove the ugliness, switching back to the multiple operators per line version.

Reply:
@cabo Do you know a way of avoiding these extraneous double quotes?

Reply:
The extra quotes are added by xml2rfc, but I don't know why or how to avoid this. I [posted](https://mailarchive.ietf.org/arch/msg/xml2rfc/30tTnMMcJHCIH8t8-s_NVLZYrFg/) to the xml2rfc mailing list about this.

Reply:
So it seems the extra quotes are caused by a [bug](https://trac.ietf.org/trac/xml2rfc/ticket/600) which will be fixed in the upcoming release of xml2rfc. I propose to leave them be for now and pick up the new release of xml2rfc in due course.